src/api/app/models/configuration.rb

```diff
@@ -82,13 +82,16 @@ def amqp_namespace
     CONFIG['amqp_namespace'] || 'opensuse.obs'
   end
 
-  def passwords_changable?
-    change_password && CONFIG['proxy_auth_mode'] != :on && CONFIG['ldap_mode'] != :on
+  def passwords_changable?(user = nil)
+    change_password && CONFIG['proxy_auth_mode'] != :on && (user.try(:ignore_auth_services?) || CONFIG['ldap_mode'] != :on)
   end
 
-  def accounts_editable?
-    (CONFIG['proxy_auth_mode'] != :on || (CONFIG['proxy_auth_mode'] == :on && CONFIG['proxy_auth_account_page'].present?)) && \
-      CONFIG['ldap_mode'] != :on
+  def accounts_editable?(user = nil)
+    (
+      CONFIG['proxy_auth_mode'] != :on || CONFIG['proxy_auth_account_page'].present?
```

Comment:
Why did you change
```
(CONFIG['proxy_auth_mode'] == :on && CONFIG['proxy_auth_account_page'].present?)
```

Reply:
@ChrisBr Because the second check (`|| CONFIG['proxy_auth_mode'] == :on `) was redundant. If 'proxy_auth_mode' is not :on, we can assume it's :on.